google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/spi/v1/SubscriberImplTest.java

```diff
@@ -276,8 +277,8 @@ public void testModifyAckDeadline() throws Exception {
     Subscriber subscriber =
         startSubscriber(
             getTestSubscriberBuilder(testReceiver)
-                .setAckExpirationPadding(Duration.standardSeconds(1)));
-
+                .setAckExpirationPadding(Duration.standardSeconds(1))
+                .setMaxAckExtensionPeriod(Duration.standardSeconds(13)));
```

Comment:
It seems like we should have another test where MaxAckExtensionPeriod is not set.